test/Container/RoleServiceFactoryTest.php

```diff
@@ -18,69 +18,81 @@
 
 namespace ZfcRbacTest\Container;
 
+use Rbac\Rbac;
+use Rbac\Traversal\Strategy\TraversalStrategyInterface;
 use Zend\ServiceManager\ServiceManager;
-use ZfcRbac\Factory\RoleServiceFactory;
+use ZfcRbac\Container\RoleServiceFactory;
+use ZfcRbac\Exception\RuntimeException;
+use ZfcRbac\Identity\AuthenticationProvider;
+use ZfcRbac\Identity\IdentityProviderInterface;
 use ZfcRbac\Options\ModuleOptions;
 use ZfcRbac\Role\RoleProviderPluginManager;
 
 /**
- * @covers \ZfcRbac\Factory\RoleServiceFactory
+ * @covers \ZfcRbac\Container\RoleServiceFactory
  */
 class RoleServiceFactoryTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @markTestSkipped skipped
+     */
     public function testFactory()
     {
+        $this->markTestSkipped(
+            'Seems Rbac\Traversal\Strategy\TraversalStrategyInterface has been removed. Not sure what this does.'
+        );
+
         $options = new ModuleOptions([
-            'identity_provider'    => 'ZfcRbac\Identity\AuthenticationProvider',
+            'identity_provider'    => AuthenticationProvider::class,
             'guest_role'           => 'guest',
             'role_provider'        => [
-                'ZfcRbac\Role\InMemoryRoleProvider' => [
+                \ZfcRbac\Role\InMemoryRoleProvider::class => [
                     'foo'
                 ]
             ]
         ]);
 
         $serviceManager = new ServiceManager();
-        $serviceManager->setService('ZfcRbac\Options\ModuleOptions', $options);
-        $serviceManager->setService('ZfcRbac\Role\RoleProviderPluginManager', new RoleProviderPluginManager());
+        $serviceManager->setService(ModuleOptions::class, $options);
+        $serviceManager->setService(RoleProviderPluginManager::class, new RoleProviderPluginManager($serviceManager));
         $serviceManager->setService(
-            'ZfcRbac\Identity\AuthenticationProvider',
-            $this->getMock('ZfcRbac\Identity\IdentityProviderInterface')
+            AuthenticationProvider::class,
+            $this->getMock(IdentityProviderInterface::class)
         );
 
-        $traversalStrategy = $this->getMock('Rbac\Traversal\Strategy\TraversalStrategyInterface');
-        $rbac              = $this->getMock('Rbac\Rbac', [], [], '', false);
+        $traversalStrategy = $this->getMock(TraversalStrategyInterface::class);
```

Comment:
You're right, TraversalStraetgyInterface has been removed from Rbac, that's something that will need to be updated in RoleService, but let's do that after :).